src/utils/conf.rs

```diff
@@ -35,6 +35,7 @@ pub struct Conf {
     pub data_directory: PathBuf,
     pub run_indexer: bool,
     pub da_address: String,
+    pub tcp_server_address: String,
```

Comment:
maybe this could be optional, and if not present (default), don't run the tcp server ?